docs/Rakefile

```diff
@@ -4,6 +4,9 @@ require 'rubygems/package'
 task :default => :gen
 
 BUILD_DIR          = "build"
+DITAA_IMAGES       = [
```

Comment:
-1 This does not feel right and it is also fragile.

Reply:
We discussed this before. This is problematic out of two reasons:

* It requires you to know that there is a special case. What if we add a ditaa image into another topic? Then things will break again (provided that we are actually fixing a problem here which I doubt)
* Even more problematic is that all images are excluded. What if someone wants to add a static image to the topic. Locally it all would render fine, but the image would disappear when the docs would get build as part of the openshift.org build via the tarball.